ovos_audio/playback.py

```
@@ -1,12 +1,17 @@
 import random
+from queue import Empty
+from threading import Thread, Event
+from typing import Optional
+
 from ovos_audio.transformers import TTSTransformersService
 from ovos_bus_client.message import Message
+from ovos_bus_client.util import get_message_lang
 from ovos_config import Configuration
+from ovos_plugin_manager.g2p import OVOSG2PFactory, find_g2p_plugins
```

Ruff

`ovos_plugin_manager.g2p.find_g2p_plugins` imported but unused

Remove unused import: `ovos_plugin_manager.g2p.find_g2p_plugins`

(F401)